redirects-list.json

```diff
@@ -58,6 +58,8 @@
   "^/doc/use-cases/shared-development-server$                                             /doc/use-cases/fast-data-storage-layer#example-shared-development-server",
   "^/doc/use-cases/data-registries$                                                       /doc/use-cases/data-registry",
 
+  "^/doc/user-guide/what-is-dvc$                                                          /doc/user-guide/overview 302",
+  "^/doc/user-guide/related-technologies$                                                 /doc/user-guide/overview 302",
```

Comment:
302s for now in case we later decide to move it to the US index page instead, though.